.gitignore

```diff
@@ -1,39 +1,28 @@
 # General
-*
-!*/
-!*.txt
+  *
+  !*/
```

Comment:
What does this line do? Why would we want to include everything ending in a ','?

Reply:
I do not quite understand what you mean about files ending in "," being included. Could you please provide further information like line numbers please?

Have added c and asm file extensions.